db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+user = User.create!({
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name,
+    email: "example@email.com",
+    password: "password_digest"
+  })
+
+100.times do
+  Obituary.create!({
+    first_name: Faker::Name.first_name,
+    last_name: Faker::Name.last_name,
+    age: Faker::Number.within(range: 45..110),
+    city: Faker::Address.city,
+    state: Faker::Address.state_abbr,
+    description: Faker::Lorem.paragraph(sentence_count: 15),
+    image_url: Faker::Fillmurray.unique.clear,
+    user_id: user.id
+  })
+
+  50.times do
+    Obituary.create!({
+      first_name: Faker::Name.first_name,
+      last_name: Faker::Name.last_name,
+      age: Faker::Number.within(range: 45..110),
+      city: Faker::Address.city,
+      state: Faker::Address.state_abbr,
+      description: Faker::Lorem.paragraph(sentence_count: 15),
+      image_url: Faker::Fillmurray.unique.clear,
+      covid: true,
+      user_id: user.id
+    })
+  end
+end
```

Comment:
Thanks for setting up these seeds. It would be cool to add some right before codefair so our project looks awesome.

Reply:
100% agree! Once we start working with location search, we can tweak the faker seed data as needed so it shows real US cities. I think right now it is nonsense.

Reply:
I changed the image faker to try and get unique images but actually I broke it so we are back to the same picture of bill murray as placeholder for every obit.